#! /bin/sh /usr/share/dpatch/dpatch-run
## ares extra param.dpatch by  <robin@debian>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: ares extra param

@DPATCH@

+++ ./indra/llcommon/llares.cpp	2007-11-27 18:49:20.000000000 +0000
--- ./indra/llcommon/llares.cpp	2007-11-20 12:52:27.000000000 +0000
@@ -117,7 +117,7 @@
 	ares_cancel(chan_);
 }
 
+static void host_callback(void *arg, int status,int timeouts, struct hostent *ent)
-static void host_callback(void *arg, int status, struct hostent *ent)
 {
 	LLPointer<LLAres::HostResponder> *resp =
 		(LLPointer<LLAres::HostResponder> *) arg;
@@ -380,7 +380,7 @@
 			<< LLAres::strerror(code) << llendl;
 }
 
+static void nameinfo_callback(void *arg, int status,int timeouts, char *node, char *service)
-static void nameinfo_callback(void *arg, int status, char *node, char *service)
 {
 	LLPointer<LLAres::NameInfoResponder> *resp =
 		(LLPointer<LLAres::NameInfoResponder> *) arg;
@@ -402,7 +402,7 @@
 					 new LLPointer<NameInfoResponder>(resp));
 }
 
+static void search_callback(void *arg, int status,int timeouts, unsigned char *abuf,
-static void search_callback(void *arg, int status, unsigned char *abuf,
 							int alen)
 {
 	LLPointer<LLAres::QueryResponder> *resp =
